mp/src/game/shared/neo/weapons/weapon_neobasecombatweapon.h

```diff
@@ -221,7 +221,7 @@ class CNEOBaseCombatWeapon : public CBaseHL2MPCombatWeapon
 	virtual float GetFireRate() override final;
 	virtual bool GetRoundChambered() const { return 0; }
 	virtual bool GetRoundBeingChambered() const { return 0; }
-	float GetPenetration();
+	const float GetPenetration();
```

Comment:
I meant this should be declared as `float GetPenetration() const`, because it doesn't mutate the object's state. I don't think we need to return a `const float`, because it's passed by value anyway.